service/tx/network_delegation.go

```diff
@@ -109,6 +109,38 @@ func (s *Service) WithdrawDelegRewards(args client.WithdrawDelegRewardsRequest,
 	return nil
 }
 
+func (s *Service) ReinvestDelegRewards(args client.ReinvestDelegRewardsRequest, reply *client.CreateTxReply) error {
+	invest := nwd.Reinvest{
+		Delegator: args.Delegator,
+		Amount:    args.Amount,
+	}
+
+	data, err := invest.Marshal()
+	if err != nil {
+		return err
+	}
+
+	uuidNew, _ := uuid.NewUUID()
+	feeAmount := s.feeOpt.MinFee()
+	tx := &action.RawTx{
+		Type: action.REWARDS_REINVEST_NETWORK_DELEGATE,
+		Data: data,
+		Fee: action.Fee{
+			Price: action.Amount{Currency: "OLT", Value: *feeAmount.Amount},
+			Gas:   80000,
```

Comment:
Just curious ,does the trasaction fail with not enough gas ,for values less than 80000  ?

Reply:
> Just curious ,does the trasaction fail with not enough gas ,for values less than 80000 ?

I don't know the exact number of min gas that cause a failure, 80000 is an estimated gas big enough to guarantee a success.

Reply:
I failed at 40000 for some txs before